examples/linear-model.py

```diff
@@ -127,17 +127,31 @@
 # Construct the model
 # -------------------
 #
-# Before we fit the model we have to define our regression values.
+# We first initilize the :class:`equisolve.numpy.models.linear_model.Ridge`
+# object. A mandatory parameter are the ``parameter_keys`` determining with
+# respect to which parameters the regression or fit is
+# performed. Here, we choose a regression wrt. to ``"values"`` (energies) and
+# ``"positions"`` (forces).
+
+
+clf = Ridge(parameter_keys=["values", "positions"])
```

Comment:
Quick question: this parameter_keys is here to allow training of a model using only gradients, right? otherwise it feels a bit redundant: one could say we use everything defined on the input tensormap